### PR TITLE
#168055703 Fix paginator count response

### PR DESCRIPTION
**Pull request by Jonathan4github**
#### What does this PR do?
This pull request fixes the bug for paginator count. When there is a query of data it returns the total count of data in the database instead of the count of query data.
#### Description of Task to be completed
-  fix paginator count to ensure it returns the queried data count
- add ...otherData arguments in the findAndCountAll method
#### How should this be manually tested?
 - clone this repository
- ```cd``` into the repo and run ```npm install``` && ```npm run test```
#### What are the relevant pivotal tracker stories?
[#168055703](https://pivotaltracker.com/story/show/168055703)

#### Screenshots (if appropriate)
page 1
page 2
page 3

### CODE REVIEW

**Comment by iverenshaguy** [comment]
Please fix the article response message. It should be something like `Articles fetched successfully`.

File: tests/routes/articles.test.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ describe('Articles API', () => {
       const response = await chai.request(app).get(`${baseRoute}/articles?author=Lagosplk`);
 
       response.should.have.status(404);
-      response.body.message.should.equal('Lagosplk Not found');
+      response.body.error.message.should.equal('No articles found');
```
Comment (iverenshaguy) [inline tests/routes/articles.test.js]: This response should be an object that has data as an empty array, not an object that has an error message. Am I missing something?

**Review by iverenshaguy — CHANGES_REQUESTED** [review]
Good work. Please review my comments

File: tests/routes/articles.test.js (hunk 1 above)
Comment (Jonathan4github) [reply tests/routes/articles.test.js]: From the response body, it was returning an error message of 'No articles found' but I just had synced with Nenny so she told me there is no need of the test as she has updated her controller. So I will be removing this test.

**Comment by iverenshaguy** [comment]
> Please fix the article response message. It should be something like `Articles fetched successfully`.

@Jonathan4github Has this been done?

File: src/controllers/article.js [hunk 2]
```diff
@@ -517,7 +517,7 @@ export default {
     });
 
     return response.status(200).json({
-      status: 'success', data: articles, count, page, message: 'Articles fetch was succesfully'
+      status: 'success', data: articles, count, page, message: 'Articles fetch succesfully'
```
Comment (iverenshaguy) [inline src/controllers/article.js]:
```suggestion
      status: 'success', data: articles, count, page, message: 'Articles fetched successfully'
```

File: tests/routes/getAllAuthors.test.js [hunk 3]
```diff
@@ -45,9 +45,8 @@ describe('Authors Endpoint', () => {
         .get(`${baseRoute}/authors/`)
         .set('Authorization', `Bearer ${userToken}`);
 
-      response.should.have.status(404);
-      response.body.should.be.a('Object');
-      response.body.should.have.property('error');
+      response.should.have.status(200);
+      response.body.data.should.be.a('Array');
```
Comment (iverenshaguy) [inline tests/routes/getAllAuthors.test.js:49]: What data are we expecting?

**Review by iverenshaguy — CHANGES_REQUESTED** [review]

File: tests/routes/getAllAuthors.test.js (hunk 3 above)
Comment (Jonathan4github) [reply tests/routes/getAllAuthors.test.js:49]: Should return an empty array. I just implemented that.

**Review by iverenshaguy — APPROVED** [review]

**Pull request merged**